src/typingsInstall.ts

```diff
@@ -0,0 +1,12 @@
+const spawn: any = require('cross-spawn');
+
+export default async function () {
+	return new Promise((resolve, reject) => {
+		spawn('typings', ['install'], { stdio: 'inherit' })
```

Comment:
I would always favour using a programmatic API from a tool like this if one exists (personally)